app/javascript/components/profile-description.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div>
+    <div
+      v-if="user.description"
+      class="profile-info__description"
+    >
+      <div
+      v-if="!readMoreActivated && !editDescription"
+      >
+        <p
+          @dblclick="edit"
+        >
+          {{ user.description.slice(0,110) }} ...
+        </p>
+        <button
+          @click="activateReadMore"
+          class="btn-description"
+        >
+          Ver más
+        </button>
+      </div>
+      <div
+      v-if="readMoreActivated & !editDescription"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 8 spaces but found 6 spaces  vue/html-indent